packages/web3-eth-accounts/src/account.ts

```diff
@@ -0,0 +1,70 @@
+import { utils, getPublicKey } from 'ethereum-cryptography/secp256k1';
+import { toChecksumAddress, bytesToHex, sha3Raw, HexString } from 'web3-utils';
+import { InvalidPrivateKeyError, PrivateKeyLengthError } from './errors';
+
+// Will be added later
+export const encrypt = (): boolean => true;
+
+// Will be added later
+export const sign = (): boolean => true;
+
+// Will be added later
+export const signTransaction = (): boolean => true;
+
+/**
+ * Get account from private key
+ */
+export const privateKeyToAccount = (
+	privateKey: string | Buffer,
+): {
+	address: string;
+	privateKey: string;
+	signTransaction: () => boolean; // From 1.x
+	sign: () => boolean;
+	encrypt: () => boolean;
+} => {
+	if (!privateKey) {
+		throw new InvalidPrivateKeyError(privateKey);
+	}
```

Comment:
The current validation allows false positives, I think we should create `isValidStringInput` and `isBuffer` methods that return a boolean and use them for validation here. After those methods are created we could revise this to:

```typescript
if (!isValidStringInput(privateKey) || !isBuffer(privateKey)) throw new InvalidPrivateKeyError(privateKey);
```

Even though these validation methods are simple, declaring them once and reusing them across the library gives us a single method to update if we discover a bug or want to make the validation stricter. It also reduces the opportunity for accidental less/more strict checks in various parts of the codebase

FYI the new `isBuffer` method (that would be added to `web3-utils/src/validation.ts`) would use: [Buffer.isBuffer](https://www.w3schools.com/nodejs/met_buffer_isbuffer.asp)